livechat/agent/rtm/base.py

```diff
@@ -0,0 +1,43 @@
+''' Agent RTM client implementation. '''
+
+# pylint: disable=W0613,W0622,C0103,R0913,R0903,W0107,W0221
+from __future__ import annotations
+
+from typing import Union
+
+from livechat.agent.rtm.api.v33 import AgentRTMV33
+from livechat.agent.rtm.api.v34 import AgentRTMV34
+from livechat.agent.rtm.api.v35 import AgentRTMV35
+from livechat.config import CONFIG
+
+stable_version = CONFIG.get('stable')
+api_url = CONFIG.get('url')
+
+
+class AgentRTM:
+    ''' Main class that gets specific client. '''
+    @staticmethod
+    def get_client(
+        version: str = stable_version,
+        base_url: str = api_url
+    ) -> Union[AgentRTMV33, AgentRTMV34, AgentRTMV35]:
+        ''' Returns client for specific Agent RTM version.
+
+            Args:
+                version (str): API's version. Defaults to the stable version of API.
+                base_url (str): API's base url. Defaults to API's production URL.
+
+            Returns:
+                AgentRTMInterface: API client object for specified version.
```

Comment:
```suggestion
                AgentRTM: API client object for specified version.
```

Reply:
I'll remove the type from the docstring so it's not misleading, we have annotations for a reason